storops/unity/resource/cg.py

```diff
@@ -148,7 +148,7 @@ def modify_lun(self, lun, name=None, size=None, host_access=None,
             sp=sp,
             io_limit_policy=io_limit_policy,
             tiering_policy=tiering_policy,
-            is_compression=is_compression)
+            is_data_reduction_enabled=is_data_reduction_enabled)
```

Comment:
should pass in the `cli` to `prepare_lun_parameters`.

Reply:
Fixed.